agenta-cli/agenta/api/api.py

```diff
@@ -0,0 +1,82 @@
+import os
+import toml
+import time
+import click
+from typing import Dict
+from pathlib import Path
+from agenta.api.backend import client
+from agenta.client.api_models import Image
+from agenta.api.exceptions import APIRequestError
+from requests.exceptions import RequestException
+
+
+class ClientWrapper:
```

Comment:
why do we have the clientwrapper? I don't seen any logic inside other than adding a timeout???